applications_tests/dem_3d/torque_boundary_box.output

```diff
@@ -0,0 +1,43 @@
+DEM time-step is 4.64363% of Rayleigh time step
+*****************************************************************
+Starting simulation with Lethe/DEM on 1 processors
+*****************************************************************
+
+*****************************************************************
+10 particles of type 0 were inserted, 0 particles of type 0 remaining
+*****************************************************************
+Boundary 0 :
+Force = 0 0 0
+Moment = 0 0 0
+
+Boundary 1 :
+Force = 0 0 0
+Moment = 0 0 0
+
+Boundary 2 :
+Force = 0 0 0
+Moment = 0 0 0
+
+Boundary 3 :
+Force = 0 0 0
+Moment = 0 0 0
+
+Boundary 4 :
+Force = 0 0 -0.01309
+Moment = 0 -0.0005318 0
```

Comment:
I am really not a fan of this output format, but this can be modified later.
I would have imagined something like
Boundary_id  f_x f_y f_z t_x t_y t_z
0 x x x x x
1 x x x x x

etc.

Reply:
Yeah std:setw is doable. Do i do it in this prm? @blaisb

Reply:
I'd suggest using a TableHandler like is done in the other force output of Lethe instead of this approach. This can be done in a next PR.